### PR TITLE
Added continents.

### PR DESCRIPTION
**Pull request by marvinIsSacul**
I added continents.

### CODE REVIEW

File: lib/src/data/address/continents.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  'Africa',
+  'Antarctica',
+  'Asia',
+  'Europe',
+  'North America',
+  
+  // Some people use austrailia, whilst others use oceania.
+  'Oceania',
+
+  'South Ameria',
```
Comment (drager) [inline lib/src/data/address/continents.dart]:
```suggestion
  'South America',
```

File: test/specs/address.dart [hunk 2]
```diff
@@ -54,5 +54,9 @@ main() {
     test('should be able to generate country code', () {
       expect(faker.address.countryCode(), matches(new RegExp(r'^[A-Z]{2}$')));
     });
+
+    test('should be able to generate continent', () {
+      expect(faker.address.neighborhood(), matches(new RegExp(r'^[\w\s]*$')));
```
Comment (drager) [inline test/specs/address.dart]: This test looks like it's calling the wrong function.

File: lib/src/address.dart [hunk 3]
```diff
@@ -117,4 +118,12 @@ class Address {
   ///   faker.address.countryCode();
   /// ```
   String countryCode() => random.element(countryCodes);
+
+  /// Generates a country code.
+  ///
+  /// Example:
+  /// ```dart
+  ///   faker.address.countryCode();
```
Comment (drager) [inline lib/src/address.dart]: Please update the doc comments.

**Review by drager — CHANGES_REQUESTED** [review]
Thank you so much for this!

I added some comments that needs to be resolved. Also, consider running `dartfmt` on the code.

**Comment by marvinIsSacul** [comment]
Oh, sorry. I didn't notice those errors. 

But I've fixed them now. Thanks.

And format problems are probably caused by my tab sizes setting. I've set my tab size to 2 spaces.

**Review by drager — APPROVED** [review]
Thanks @marvinIsSacul! This looks great!

**Comment by marvinIsSacul** [comment]
Thanks.

**Pull request merged**